### PR TITLE
Feature/bookmark businesses

### PR DESCRIPTION
**Pull request by AliNooner**
## Type of change

- [x] New feature
- [ ] Bug fix
- [ ] Refactor
- [ ] Documentation update
- [ ] Testing

## Description

Add ability for user to be able to click on a Bookmark button and add a listing they would like to visit to a Bookmark page.

## Requested Feedback

Any possible refactors. 

## Checklist:

- [x] I have linked the appropriate Project Board
- [x] I have linked any issues this closes
- [x] My code has no unused/commented out code
- [x] I have performed a self-review of my code
- [x] I have fully tested my code

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+import { BookmarkContextProvider } from '../../contexts/BookmarkContext'
 import { Routes, Route, NavLink } from 'react-router-dom'
 import Form from '../Form/Form'
 
 const App = () => {
   return (
     <LocationContextProvider>
       <BusinessContextProvider>
+
+      <BookmarkContextProvider>
```
Comment (kmewrd) [inline src/components/App/App.js:17]: I know this is minor, but could we fix indentation here? It looks like the `<BookmarkContextProvider>` isn't nested down.

File: src/contexts/BookmarkContext.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { createContext, useState, useContext } from "react";
+import { BusinessContext  } from "./BusinessContext";
+
+const BookmarkContext = createContext([])
+
+const BookmarkContextProvider = ({ children }) => {
+  const [bookmarked, setBookmarked] = useState([])
```
Comment (kmewrd) [inline src/contexts/BookmarkContext.js:7]: Can we change this variable name from bookmarked to bookmarks? I feel like that's more semantic for what this array holds - bookmarks. :)

Comment (AliNooner) [reply src/contexts/BookmarkContext.js]: Done and done! Thanks Kim

**Comment by kmewrd** [comment]
Awesome, thanks for working on that! At a later date we might want to add some conditional rendering for the button so that is says "Un-bookmark" if the business has already been bookmarked. (If we have time!)

**Pull request merged**